ddapm_test_agent/agent.py

```diff
@@ -895,14 +920,19 @@ async def handle_session_clear(self, request: Request) -> web.Response:
                 if req.match_info.handler == self.handle_session_start:
                     if _session_token(req) == session_token:
                         in_token_sync_session = True
+                        continue  # Don't clear the session start
```

Comment:
I wonder if we should add a way to maintain this functionality, `handle_session_delete` or something. However, I don't think there's actually a use-case for it so I'm fine leaving it until someone needs something like that.

Reply:
Yeah, the functional behavior is preserved (no traces/snapshots will be returned), but there is a potential issue with memory space.  

I'd prefer to leave it for now